Jenkinsfile

```diff
@@ -112,136 +101,223 @@ pipeline {
                 }
             }
         }
-        stage('Load Firmware') {
-            matrix {
-                axes {
-                    axis {
-                        name 'PROTOCOL'
-                        values 'soem', 'canopen'
+        stage('Ecat tests') {
+            options {
+                lock(ECAT_NODE_LOCK)
+            }
+            agent {
+                label ECAT_NODE
+            }
+            stages {
+                stage("Checkout") {
+                    steps {
+                        checkout scm
+                        script {installTox ()}
```

Comment:
I don't understand why we are doing all these changes here and locking the machine also during package installation instead of fixing ingenialink

Reply:
Because it is not only ingenialink, we'd change all the other pipelines that lock the machine in the options.

Reply:
https://novantamotion.atlassian.net/wiki/spaces/SKB/pages/553189390/Locking+machines+with+real+drives